packages/wallet-ui/src/store/user/user.ts

```diff
@@ -158,6 +139,7 @@ export const loginUser = (
 };
 
 export const getUser = (
+  // @ts-ignore
```

Comment:
Same as the [previous case](https://github.com/counterfactual/monorepo/pull/2102/files/5e32dca3d7565bc35ac8e2978fdf1547dd478cf1..ed9ef3dd5129d2dd9bde0aa898b36ee066a078d2#diff-1938184ada6ff6d9c1e9c95824eb2ed6R100).